core/src/main/python/wlsdeploy/tool/deploy/jms_resources_deployer.py

```diff
@@ -151,10 +154,16 @@ def _add_jndi_properties(self, property_name_nodes, location):
         name_attribute = self.alias_helper.get_wlst_attribute_name(properties_location, KEY)
         mbean_type = self.alias_helper.get_wlst_mbean_type(properties_location)
 
+        for property_name in property_name_nodes:
+            if deployer_utils.is_delete_name(property_name):
+                name = deployer_utils.get_delete_item_name(property_name)
```

Comment:
Just curious. Is WLST nice enough to remove the deleted server from all targets that reference it?

Reply:
I tried this by removing a server that is referenced as a target.  Offline, I got a pretty good error about wlst.set(Target, m2).  Offline, it has a better message, because we attempt to look up the server and don't find it.

Reply:
Did you see the comment at top of file? Related to this.